### ISSUE TITLE
Red color of tabs is ugly when using TabMixPlus

### ISSUE THREAD

**Issue by philnolan3d** [Quicksaver/Tab-Groups#243]
I assume this color is to mark unloaded tabs but it's really awful and kind of hurts the eyes to look at. Is there some way it could be toned down to a pale color like the tan that TabMixPlus uses or a pastel like pink even?

![tabgroups_red](https://cloud.githubusercontent.com/assets/19702736/15735937/54f8d468-286a-11e6-9589-3311674bd159.jpg)

**Comment by Quicksaver** [comment]
I don't understand what you mean by "tan that TMP uses", it uses the exact same color as defined in TMP's preferences.

But perhaps it can be made more subtle.

**Comment by philnolan3d** [comment]
Sorry I wasn't clear about the tan color. I meant the actual tabs that are not loaded. L
![tan_tabs](https://cloud.githubusercontent.com/assets/19702736/15738854/5436ba5c-287b-11e6-9935-981614e6e136.jpg)
ike in this pic.

**Comment by Quicksaver** [comment]
Oh, I don't believe that color would be of much use, notice it's almost the same as the background of the groups. But it's not a bad suggestion if you change from the default value though, I'll definitely consider it.

**Comment by ng110** [comment]
I agree that the red borders are really ugly and painful to look at.  Can we not have just the text in red/italic (as they are), and the borders looking normal?

Cheers!
